{"url": "https://api.github.com/repos/tensorflow/tensorflow/issues/3273", "repository_url": "https://api.github.com/repos/tensorflow/tensorflow", "labels_url": "https://api.github.com/repos/tensorflow/tensorflow/issues/3273/labels{/name}", "comments_url": "https://api.github.com/repos/tensorflow/tensorflow/issues/3273/comments", "events_url": "https://api.github.com/repos/tensorflow/tensorflow/issues/3273/events", "html_url": "https://github.com/tensorflow/tensorflow/issues/3273", "id": 164914891, "node_id": "MDU6SXNzdWUxNjQ5MTQ4OTE=", "number": 3273, "title": "Cannot build from source", "user": {"login": "BernardoGO", "id": 1606650, "node_id": "MDQ6VXNlcjE2MDY2NTA=", "avatar_url": "https://avatars2.githubusercontent.com/u/1606650?v=4", "gravatar_id": "", "url": "https://api.github.com/users/BernardoGO", "html_url": "https://github.com/BernardoGO", "followers_url": "https://api.github.com/users/BernardoGO/followers", "following_url": "https://api.github.com/users/BernardoGO/following{/other_user}", "gists_url": "https://api.github.com/users/BernardoGO/gists{/gist_id}", "starred_url": "https://api.github.com/users/BernardoGO/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/BernardoGO/subscriptions", "organizations_url": "https://api.github.com/users/BernardoGO/orgs", "repos_url": "https://api.github.com/users/BernardoGO/repos", "events_url": "https://api.github.com/users/BernardoGO/events{/privacy}", "received_events_url": "https://api.github.com/users/BernardoGO/received_events", "type": "User", "site_admin": false}, "labels": [], "state": "closed", "locked": false, "assignee": null, "assignees": [], "milestone": null, "comments": 1, "created_at": "2016-07-11T19:27:58Z", "updated_at": "2016-07-13T20:21:52Z", "closed_at": "2016-07-13T20:21:52Z", "author_association": "NONE", "body_html": "<p>Cannot build from source</p>\n<h3>Environment info</h3>\n<p>Operating System: Arch Linux</p>\n<p>Installed version of CUDA and cuDNN:<br>\n7.5</p>\n<p>(please attach the output of <code>ls -l /path/to/cuda/lib/libcud*</code>):<br>\n-rwxr-xr-x 1 root root 311596 May  1 07:04 /opt/cuda/lib/libcudart.so.7.5.18<br>\n-rw-r--r-- 1 root root 558020 May  1 07:04 /opt/cuda/lib/libcudart_static.a</p>\n<p>If installed from binary pip package, provide:</p>\n<ol>\n<li>Which pip package you installed.</li>\n<li>The output from <code>python -c \"import tensorflow; print(tensorflow.__version__)\"</code>.<br>\n0.9.0rc0</li>\n</ol>\n<p>If installed from sources, provide the commit hash:<br>\n<a class=\"commit-link\" data-hovercard-type=\"commit\" data-hovercard-url=\"https://github.com/tensorflow/tensorflow/commit/ecdb06c03db58daef5a8271bdef9f85c3bf28d08/hovercard\" href=\"https://github.com/tensorflow/tensorflow/commit/ecdb06c03db58daef5a8271bdef9f85c3bf28d08\"><tt>ecdb06c</tt></a></p>\n<h3>Steps to reproduce</h3>\n<ol>\n<li>bazel build -c opt --config=cuda --verbose_failures //tensorflow/tools/pip_package:build_pip_package</li>\n</ol>\n<h3>What have you tried?</h3>\n<ol>\n<li>bazel build -c opt --genrule_strategy=standalone --spawn_strategy=standalone --config=cuda --verbose_failures //tensorflow/tools/pip_package:build_pip_package</li>\n</ol>\n<h3>Logs or other output that would be helpful</h3>\n<p>(If logs are large, please upload as attachment).</p>\n<pre><code>ERROR: /home/bernardo/.cache/bazel/_bazel_bernardo/6e092ee851f20b3b3e86c2bd0d1d1759/external/protobuf/BUILD:331:1: C++ compilation of rule '@protobuf//:protoc' failed: crosstool_wrapper_driver_is_not_gcc failed: error executing command \n  (cd /home/bernardo/.cache/bazel/_bazel_bernardo/6e092ee851f20b3b3e86c2bd0d1d1759/tensorflow &amp;&amp; \\\n  exec env - \\\n    PATH=/opt/cuda/bin/:/usr/local/sbin:/usr/local/bin:/usr/bin:/snap/bin:/opt/cuda/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl \\\n  third_party/gpus/crosstool/clang/bin/crosstool_wrapper_driver_is_not_gcc -U_FORTIFY_SOURCE '-D_FORTIFY_SOURCE=1' -fstack-protector -fPIE -Wall -Wunused-but-set-parameter -Wno-free-nonheap-object -fno-omit-frame-pointer -g0 -O2 -DNDEBUG -ffunction-sections -fdata-sections -g0 '-std=c++11' -iquote external/protobuf -iquote bazel-out/host/genfiles/external/protobuf -iquote external/bazel_tools -iquote bazel-out/host/genfiles/external/bazel_tools -isystem external/protobuf/src -isystem bazel-out/host/genfiles/external/protobuf/src -isystem external/bazel_tools/tools/cpp/gcc3 -no-canonical-prefixes -Wno-builtin-macro-redefined '-D__DATE__=\"redacted\"' '-D__TIMESTAMP__=\"redacted\"' '-D__TIME__=\"redacted\"' -fno-canonical-system-headers '-frandom-seed=bazel-out/host/bin/external/protobuf/_objs/protoc/external/protobuf/src/google/protobuf/compiler/main.o' -MD -MF bazel-out/host/bin/external/protobuf/_objs/protoc/external/protobuf/src/google/protobuf/compiler/main.d -c external/protobuf/src/google/protobuf/compiler/main.cc -o bazel-out/host/bin/external/protobuf/_objs/protoc/external/protobuf/src/google/protobuf/compiler/main.o): com.google.devtools.build.lib.shell.BadExitStatusException: Process exited with status 1.\ngcc: error trying to exec 'cc1plus': execvp: No such file or directory\nTarget //tensorflow/tools/pip_package:build_pip_package failed to build\nINFO: Elapsed time: 1.540s, Critical Path: 0.24s\n</code></pre>", "body_text": "Cannot build from source\nEnvironment info\nOperating System: Arch Linux\nInstalled version of CUDA and cuDNN:\n7.5\n(please attach the output of ls -l /path/to/cuda/lib/libcud*):\n-rwxr-xr-x 1 root root 311596 May  1 07:04 /opt/cuda/lib/libcudart.so.7.5.18\n-rw-r--r-- 1 root root 558020 May  1 07:04 /opt/cuda/lib/libcudart_static.a\nIf installed from binary pip package, provide:\n\nWhich pip package you installed.\nThe output from python -c \"import tensorflow; print(tensorflow.__version__)\".\n0.9.0rc0\n\nIf installed from sources, provide the commit hash:\necdb06c\nSteps to reproduce\n\nbazel build -c opt --config=cuda --verbose_failures //tensorflow/tools/pip_package:build_pip_package\n\nWhat have you tried?\n\nbazel build -c opt --genrule_strategy=standalone --spawn_strategy=standalone --config=cuda --verbose_failures //tensorflow/tools/pip_package:build_pip_package\n\nLogs or other output that would be helpful\n(If logs are large, please upload as attachment).\nERROR: /home/bernardo/.cache/bazel/_bazel_bernardo/6e092ee851f20b3b3e86c2bd0d1d1759/external/protobuf/BUILD:331:1: C++ compilation of rule '@protobuf//:protoc' failed: crosstool_wrapper_driver_is_not_gcc failed: error executing command \n  (cd /home/bernardo/.cache/bazel/_bazel_bernardo/6e092ee851f20b3b3e86c2bd0d1d1759/tensorflow && \\\n  exec env - \\\n    PATH=/opt/cuda/bin/:/usr/local/sbin:/usr/local/bin:/usr/bin:/snap/bin:/opt/cuda/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl \\\n  third_party/gpus/crosstool/clang/bin/crosstool_wrapper_driver_is_not_gcc -U_FORTIFY_SOURCE '-D_FORTIFY_SOURCE=1' -fstack-protector -fPIE -Wall -Wunused-but-set-parameter -Wno-free-nonheap-object -fno-omit-frame-pointer -g0 -O2 -DNDEBUG -ffunction-sections -fdata-sections -g0 '-std=c++11' -iquote external/protobuf -iquote bazel-out/host/genfiles/external/protobuf -iquote external/bazel_tools -iquote bazel-out/host/genfiles/external/bazel_tools -isystem external/protobuf/src -isystem bazel-out/host/genfiles/external/protobuf/src -isystem external/bazel_tools/tools/cpp/gcc3 -no-canonical-prefixes -Wno-builtin-macro-redefined '-D__DATE__=\"redacted\"' '-D__TIMESTAMP__=\"redacted\"' '-D__TIME__=\"redacted\"' -fno-canonical-system-headers '-frandom-seed=bazel-out/host/bin/external/protobuf/_objs/protoc/external/protobuf/src/google/protobuf/compiler/main.o' -MD -MF bazel-out/host/bin/external/protobuf/_objs/protoc/external/protobuf/src/google/protobuf/compiler/main.d -c external/protobuf/src/google/protobuf/compiler/main.cc -o bazel-out/host/bin/external/protobuf/_objs/protoc/external/protobuf/src/google/protobuf/compiler/main.o): com.google.devtools.build.lib.shell.BadExitStatusException: Process exited with status 1.\ngcc: error trying to exec 'cc1plus': execvp: No such file or directory\nTarget //tensorflow/tools/pip_package:build_pip_package failed to build\nINFO: Elapsed time: 1.540s, Critical Path: 0.24s", "body": "Cannot build from source\n### Environment info\n\nOperating System: Arch Linux\n\nInstalled version of CUDA and cuDNN: \n7.5\n\n(please attach the output of `ls -l /path/to/cuda/lib/libcud*`):\n-rwxr-xr-x 1 root root 311596 May  1 07:04 /opt/cuda/lib/libcudart.so.7.5.18\n-rw-r--r-- 1 root root 558020 May  1 07:04 /opt/cuda/lib/libcudart_static.a\n\nIf installed from binary pip package, provide:\n1. Which pip package you installed.\n2. The output from `python -c \"import tensorflow; print(tensorflow.__version__)\"`.\n   0.9.0rc0\n\nIf installed from sources, provide the commit hash:\necdb06c03db58daef5a8271bdef9f85c3bf28d08\n### Steps to reproduce\n1. bazel build -c opt --config=cuda --verbose_failures //tensorflow/tools/pip_package:build_pip_package\n### What have you tried?\n1. bazel build -c opt --genrule_strategy=standalone --spawn_strategy=standalone --config=cuda --verbose_failures //tensorflow/tools/pip_package:build_pip_package\n### Logs or other output that would be helpful\n\n(If logs are large, please upload as attachment).\n\n```\nERROR: /home/bernardo/.cache/bazel/_bazel_bernardo/6e092ee851f20b3b3e86c2bd0d1d1759/external/protobuf/BUILD:331:1: C++ compilation of rule '@protobuf//:protoc' failed: crosstool_wrapper_driver_is_not_gcc failed: error executing command \n  (cd /home/bernardo/.cache/bazel/_bazel_bernardo/6e092ee851f20b3b3e86c2bd0d1d1759/tensorflow && \\\n  exec env - \\\n    PATH=/opt/cuda/bin/:/usr/local/sbin:/usr/local/bin:/usr/bin:/snap/bin:/opt/cuda/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl \\\n  third_party/gpus/crosstool/clang/bin/crosstool_wrapper_driver_is_not_gcc -U_FORTIFY_SOURCE '-D_FORTIFY_SOURCE=1' -fstack-protector -fPIE -Wall -Wunused-but-set-parameter -Wno-free-nonheap-object -fno-omit-frame-pointer -g0 -O2 -DNDEBUG -ffunction-sections -fdata-sections -g0 '-std=c++11' -iquote external/protobuf -iquote bazel-out/host/genfiles/external/protobuf -iquote external/bazel_tools -iquote bazel-out/host/genfiles/external/bazel_tools -isystem external/protobuf/src -isystem bazel-out/host/genfiles/external/protobuf/src -isystem external/bazel_tools/tools/cpp/gcc3 -no-canonical-prefixes -Wno-builtin-macro-redefined '-D__DATE__=\"redacted\"' '-D__TIMESTAMP__=\"redacted\"' '-D__TIME__=\"redacted\"' -fno-canonical-system-headers '-frandom-seed=bazel-out/host/bin/external/protobuf/_objs/protoc/external/protobuf/src/google/protobuf/compiler/main.o' -MD -MF bazel-out/host/bin/external/protobuf/_objs/protoc/external/protobuf/src/google/protobuf/compiler/main.d -c external/protobuf/src/google/protobuf/compiler/main.cc -o bazel-out/host/bin/external/protobuf/_objs/protoc/external/protobuf/src/google/protobuf/compiler/main.o): com.google.devtools.build.lib.shell.BadExitStatusException: Process exited with status 1.\ngcc: error trying to exec 'cc1plus': execvp: No such file or directory\nTarget //tensorflow/tools/pip_package:build_pip_package failed to build\nINFO: Elapsed time: 1.540s, Critical Path: 0.24s\n```\n"}